docs/reference/developer_documentation.md

```diff
@@ -105,3 +105,20 @@ poetry publish --build
 1. [Download Node.js](https://nodejs.org/en/download/)
 2. [Install Angular](https://angular.io/guide/setup-local)
 3. See [buildingmotif-app/README.md](buildingmotif-app/README.md)
+
+
+## Running on Docker
+1. Set config file.
+    ```
+    cp configs.py.dist configs.py
+    echo "DB_URI = 'postgresql://buildingmotif:password@buildingmotif-db'" > configs.py
+    ```
+2. Up containers with docker-compose.
+    ```
+    docker-compose up --build
+    ```
+3. Inside `buildingmotif-api`, migrate db.
+    ```
+    docker exec -it buildingmotif-api alembic upgrade head
+    ```
```

Comment:
This is handled inside the docker container startup: https://github.com/NREL/BuildingMOTIF/blob/develop/buildingmotif/api/start.sh

Would recommend removing this line too